File: Modulo_I/Examen Practico/Agenda/contacto.php
<?php

class Usuario{


    private $nombre;
    private $apellidos;
    private $telefono;
    private $email;
    private $direccion;


    function __construct($nombre,$apellidos,$telefono,$email,$direccion){

        $this->nombre=$nombre;
        $this->apellidos=$apellidos;
        $this->telefono=$telefono;
        $this->email=$email;
        $this->direccion=$direccion;

    }

    /**
     * GET
     */

    public function getNombre()
    {
        return $this->nombre;
    }

    public function getApellidos()
    {
        return $this->apellidos;
    }

    public function getTelefono()
    {
        return $this->telefono;
    }

    public function getEmail()
    {
        return $this->email;
    }

    public function getDireccion()
    {
        return $this->direccion;
    }


    /**
     * SET
     */
    public function setNombre($nombre)
    {
        $this->nombre = $nombre;
    }

    public function setApellidos($apellidos)
    {
        $this->apellidos = $apellidos;
    }

    public function setTelefono($telefono)
    {
        $this->telefono = $telefono;
    }

    public function setEmail($email)
    {
        $this->email = $email;
    }

    public function setDireccion($direccion)
    {
        $this->direccion = $direccion;
    }
}

?>

File: Modulo_II/ProyectosJEE/AgendaSpringMongoDB/src/beans/Contacto.java
package beans;

import java.time.LocalDate;

import org.bson.types.ObjectId;
import org.springframework.data.annotation.Id;
import org.springframework.data.mongodb.core.mapping.Document;

@Document
public class Contacto {
	
	@Id
	private ObjectId _id;
	private String nombre;
	private String apellidos;
	private String dni;
	private String telefono;
	private LocalDate fecha;
	
	public Contacto(){}
	
	public Contacto(ObjectId _id, String nombre, String apellidos, String dni, String telefono, LocalDate fecha) {
		super();
		this._id = _id;
		this.nombre = nombre;
		this.apellidos = apellidos;
		this.dni = dni;
		this.telefono = telefono;
		this.fecha = fecha;
	}
	
	public ObjectId getId() {
		return _id;
	}

	public void setId(ObjectId _id) {
		this._id = _id;
	}

	public String getNombre() {
		return nombre;
	}

	public void setNombre(String nombre) {
		this.nombre = nombre;
	}

	public String getApellidos() {
		return apellidos;
	}

	public void setApellidos(String apellidos) {
		this.apellidos = apellidos;
	}

	public String getDni() {
		return dni;
	}

	public void setDni(String dni) {
		this.dni = dni;
	}

	public String getTelefono() {
		return telefono;
	}

	public void setTelefono(String telefono) {
		this.telefono = telefono;
	}

	public LocalDate getFecha() {
		return fecha;
	}

	public void setFecha(LocalDate fecha) {
		this.fecha = fecha;
	}

	
	
}
